April 8-14

```diff
@@ -0,0 +1,8 @@
+April 8 - April 14
+
+Monday: TBA (Gym + Movie Night…?)
+Tuesday:  Pizzelle's Confections (No break - 4:30-6:00PM chocolate)
+Wednesday: Botanical Garden
+Thursday: Museum of Art ($5 till 8PM)
+Friday: Pane E Vino live concert
+Saturday: VBC ballet show
```

Comment:
Make sure everyone has requested a ballet ticket.

Reply:
I got two already (don't even know why I got two but I'm fine).

Reply:
@dead-programmer we know why you got two.

Reply:
-_-

Reply:
@pranmar123 yeah for another Indian friend I know who's in a ballet team

Reply:
interesting invite krystal to the ballet and get her a ticket